guides/common/modules/con_updating-project-to-next-patch-version.adoc

```diff
@@ -4,7 +4,8 @@
 You can update your {ProjectServer} and {SmartProxyServer} to a new patch release version, such as from {ProjectVersion}.0 to {ProjectVersion}.1, by using the {Project} maintain tool.
 The patch releases are non-disruptive to your operating environment and often fast.
 
-You can update the underlying operating system.
+{ProjectServer} or {SmartProxyServer} updates should always be done using `{foreman-maintain} update`.
+Operating systems updates should be performed the same way.
```

Comment:
```suggestion
Always update {ProjectServer}, {SmartProxyServer}, or the operating system by using `{foreman-maintain} update`.
```
"Should" is very ambiguous and we should avoid it altogether. Either "do" or "don't".
How does it sound if we simplify it?